ush/sample_configs/RRFSFW/config.rrfsfw.hera.sh

```diff
@@ -0,0 +1,72 @@
+MACHINE="hera"
+ACCOUNT="fv3-cam"
+HPSS_ACCOUNT="fv3-cam"
+EXPT_SUBDIR="test_firewx"
+
+envir="rrfsfw"
+NET="rrfsfw"
+TAG="c0v00"
+MODEL="rrfsfw"
+RUN="rrfsfw"
+
+STMP="/scratch2/NCEPDEV/stmp3/Benjamin.Blake/test_firewx"
```

Comment:
@BenjaminBlake-NOAA, can you replace `Benjamin.Blake` with `${USER}` for STMP and PTMP

Reply:
@chan-hoo I actually meant to leave STMP and PTMP blank for the user to add whatever path they want, similar to config.rrfsfw.wcoss2.sh.  Would that approach be ok with you?

Reply:
sure :)

Reply:
Okay I just updated it.  :)
I left EXTRN_MDL_SOURCE_BASEDIR_ICS and EXTRN_MDL_SOURCE_BASEDIR_LBCS as one of my directories.  RRFSFW will mainly run on WCOSS2, but if we should get sample data added to the role.epic/UFS_SRW_data directory on Hera, we can go that route.